libraries/common/helpers/modal/withShowModal.js

```diff
@@ -0,0 +1,16 @@
+import { connect } from 'react-redux';
+import showModal from '@shopgate/pwa-common/actions/modal/showModal';
+
+/**
+ * @param {function} dispatch dispatch
```

Comment:
Please let's keep the code clean with code style. `function` is supposed to be written in uppercase: `Function`. It's not a primitive type. Please check you eslint setup here. It should've moaned at you about this! And please add a proper parameter description.